addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -394,6 +394,18 @@ body.editor_enable.editor_has_snippets {
                 }
             }
         }
+
+        .snippet-option-colorpicker, .snippet-option-topMenuColor {
```

Comment:
What about all the other colorpicker options ? Don't use this, add a unique class...